### PR TITLE
PMM-5735 Graph image share link broken

### PR DESCRIPTION
**Pull request by adivinho**
- [ ] https://github.com/Percona-Lab/pmm-submodules/pull/1077

### CODE REVIEW

File: ansible/playbook/tasks/update.yml [hunk 1]
```diff
@@ -19,6 +19,64 @@
       - percona-clickhouse-server
       - pmm2-client
       - dbaas-controller
+    pmm_rendering_dependencies:
```
Comment (AlekSi) [inline ansible/playbook/tasks/update.yml:22]: Do we really have to specify them all, or we can specify only core dependencies, and everything else will be installed as a dependency?

File: ansible/playbook/tasks/update.yml [hunk 2]
```diff
@@ -168,6 +231,20 @@
       command: /usr/share/percona-dashboards/fix-panels.py
       changed_when: False
 
+    - name: Check if Grafana Image Renderer plugin has already installed
+      stat: path=/var/lib/grafana/plugins/grafana-image-renderer
+      register: gird
```
Comment (AlekSi) [inline ansible/playbook/tasks/update.yml:236]: May we rename it? Looks like a typo of "grid" :)

Comment (adivinho) [reply ansible/playbook/tasks/update.yml:236]: g - grafana
r - renderer
i - image
d - directory

Have you got any proposition?

File: ansible/playbook/tasks/update.yml (hunk 1 above)
Comment (adivinho) [reply ansible/playbook/tasks/update.yml:22]: https://grafana.com/docs/grafana/latest/administration/image_rendering/#grafana-image-renderer-plugin-and-remote-rendering-service

Screenshot_20200902_124259

Comment (AlekSi) [reply ansible/playbook/tasks/update.yml:22]: That link should be in the comment then

File: ansible/playbook/tasks/update.yml (hunk 2 above)
Comment (AlekSi) [reply ansible/playbook/tasks/update.yml:230]: `image_renderer_path`

File: ansible/playbook/tasks/update.yml [hunk 3]
```diff
@@ -18,6 +18,65 @@
       - percona-clickhouse-client
       - percona-clickhouse-server
       - pmm2-client
+      - dbaas-controller
```
Comment (AlekSi) [inline ansible/playbook/tasks/update.yml:21]: Please don't add that line in this PR

File: ansible/playbook/tasks/update.yml [hunk 4]
```diff
@@ -18,6 +18,65 @@
...
+    pmm_rendering_dependencies:
```
Comment (AlekSi) [inline ansible/playbook/tasks/update.yml:22]: Please add a blank line before that line to give git a chance to resolve a merge conflict once we merge the next PR to the list above

**Review by AlekSi — CHANGES_REQUESTED** [review]
There are merge conflicts now

**Pull request closed without merging**